tor/controller.go

```diff
@@ -255,11 +261,16 @@ func (c *Controller) authenticate() error {
 func (c *Controller) getAuthCookie() ([]byte, error) {
 	// Retrieve the authentication methods currently supported by the Tor
 	// server.
-	authMethods, cookieFilePath, _, err := c.ProtocolInfo()
+	authMethods, cookieFilePath, version, err := c.ProtocolInfo()
 	if err != nil {
 		return nil, err
 	}
 
+	// Ensure that the Tor server respects the minimum version supported.
+	if err := supportsV3(version); err != nil {
```

Comment:
Rather than _always_ returning an error here, we can instead only return an error if they're trying to use v3 and have a version that's before the version it was added in.

Reply:
Fixed.